### ISSUE TITLE
Incorrect default package versions

### ISSUE THREAD

**Issue by bstoots** [chef-cookbooks/mysql#439]
### Cookbook version
8.0.1

### Chef-client version
12.12.15

### Platform Details
Ubuntu 14.04 x86_64

### Scenario:
Install MySQL 5.5.

### Steps to Reproduce:
Something like this should expose it:
```
mysql_client 'default' do
  action :create
end
```

### Expected Result:
Expecting mysql-client-5.5 to be installed.

### Actual Result:
```
        ================================================================================
        Error executing action `install` on resource 'apt_package[mysql-client-5.5]'
        ================================================================================

        Mixlib::ShellOut::ShellCommandFailed
        ------------------------------------
        Expected process to exit with [0], but received '100'
        ---- Begin output of apt-get -q -y install mysql-client-5.5=5.5.49-0ubuntu0.14.04.1 ----
        STDOUT: Reading package lists...
        Building dependency tree...
        Reading state information...
        STDERR: E: Version '5.5.49-0ubuntu0.14.04.1' for 'mysql-client-5.5' was not found
        ---- End output of apt-get -q -y install mysql-client-5.5=5.5.49-0ubuntu0.14.04.1 ----
        Ran apt-get -q -y install mysql-client-5.5=5.5.49-0ubuntu0.14.04.1 returned 100
```

As pointed out in https://github.com/chef-cookbooks/mysql/issues/436 it appears the hard coded package version values are out of date.  A short term mitigation would be to simply update them to current package version, in this case 5.5.50-0ubuntu0.14.04.1.  However it does beg the question why are these defaults set at all?  Why not just let the system package manager handle installing the most current version unless a package_version override is specified?

**Comment by wmarra** [comment]
Same scenery here, this problem occur with *mysql_client* and *mysql_service*. I'm looking for answers too, but nothing until now.

**Comment by rschmidtz** [comment]
Same problem with Centos7

**Comment by irvingpop** [comment]
The whole `default_package_versions` thing seems less than ideal for the common use case, and quickly get out of date and unusable (especially on Ubuntu).

I got around this by passing an empty string (nil didn't work) to `package_version` like so:
```ruby
mysql_service 'local' do
  port '3306'
  version '5.7'
  package_version ''
  initial_root_password 'password'
  action [:create, :start]
end
```

**Comment by someara** [comment]
Fixed up in 8.0.2
